app/src/test/kotlin/fr/free/nrw/commons/utils/imageUntilsTest.kt

```diff
@@ -0,0 +1,14 @@
+package fr.free.nrw.commons.utils
+
+import org.junit.Assert.assertEquals
+import org.junit.Test
+
+class imagecheck {
+    @Test
+    fun validateImageForOkImage() {
+        assertEquals(ImageUtils.IMAGE_OK, ImageUtils.checkIfImageIsTooDark("src/test/resources/fr/free/nrw/commons/utils/checkalgo1.png"))
```

Comment:
Instead of using `.png` files for these tests, can we use `.jpg` files (with Exif thumbnail and without), which is what real-world devices use for saving photos?

Can we also have images that are more modest in size (i.e. they don't need to be 5 MB), perhaps <1MB each.  We want to test the algorithm, not the VM memory limits.